File: app/src/main/java/com/commit451/gitlab/model/api/ArtifactsFile.java
package com.commit451.gitlab.model.api;

import com.squareup.moshi.Json;

import org.parceler.Parcel;

@Parcel
public class ArtifactsFile {

    @Json(name = "filename")
    String fileName;
    @Json(name = "size")
    long size;

    public String getFileName() {
        return fileName;
    }

    public long getSize() {
        return size;
    }
}

File: app/src/main/java/com/commit451/gitlab/model/api/ArtifactsFile.kt
package com.commit451.gitlab.model.api

import android.os.Parcelable
import com.squareup.moshi.Json
import kotlinx.android.parcel.Parcelize

@Parcelize
data class ArtifactsFile(
    @Json(name = "filename")
    var fileName: String = "",
    @Json(name = "size")
    var size: Long = 0
) : Parcelable
